core/indexer/operator_sockets_filterer.go

```diff
@@ -114,7 +114,7 @@ func (f *operatorSocketsFilterer) WatchOperatorSocketUpdate(ctx context.Context,
 	}
 
 	sink := make(chan *blsregcoord.ContractBLSRegistryCoordinatorWithIndicesOperatorSocketUpdate)
-	operatorID := []core.OperatorID{operatorId}
+	operatorID := [][32]byte{operatorId}
```

Comment:
Is this intended given the goal of this pr?

Reply:
yeah this is a consequence of making `core.OperatorID` an explicit type - contract bindings aren't aware of `core.OperatorID`, so needs to be converted to a primitive type